.github/workflows/workflow.yml

```diff
@@ -386,11 +386,20 @@ jobs:
             bash e2e-tests/cycles_burn.sh
 
   benchmark:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     needs: cargo-build
-
+    env:
+      PROJECT_DIR: ./
```

Comment:
should it be just `.`? otherwise it gets propagated to all the variables downstream and if they have leading slash it translates into double slashes. it doesn't do harm, but it looks odd

```sh
+ CANISTER_PATH=./
+ COMMENT_MESSAGE_PATH=/tmp/canbench_comment_message.txt
+ MAIN_BRANCH_DIR=_canbench_main_branch
+ CANBENCH_OUTPUT=/tmp/canbench_output.txt
+ CANBENCH_RESULTS_FILE=.//canbench_results.yml
+ MAIN_BRANCH_RESULTS_FILE=_canbench_main_branch/.//canbench_results.yml
```

Reply:
Done.